### PR TITLE
11624

### PR DESCRIPTION
**Pull request by animeshsahay**
11624

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [- ] All filenames are lower case (including folders, images, files, etc.)
- [-] Filenames are descriptive
- [-] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [-] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [-] Image references in markdown contain an alternate text

### CODE REVIEW

File: ai-chatbot-ampere/lab1_setup_vcn_compute/lab1_setup_vcn_compute.md
Comment (hope-fisher) [inline ai-chatbot-ampere/lab1_setup_vcn_compute/lab1_setup_vcn_compute.md:1]: alt text needed

File: ai-chatbot-ampere/lab2_initialize_compute_with_dependencies/lab2_initialize_compute_with_dependencies.md
Comment (hope-fisher) [inline ai-chatbot-ampere/lab2_initialize_compute_with_dependencies/lab2_initialize_compute_with_dependencies.md:1]: alt text needed on images

File: ai-chatbot-ampere/lab3_setup_container_registory/lab3_setup_container_registory.md
Comment (hope-fisher) [inline ai-chatbot-ampere/lab3_setup_container_registory/lab3_setup_container_registory.md:1]: alt text needed on images

File: ai-chatbot-ampere/lab4_deploy_application/lab4_deploy_application.md
Comment (hope-fisher) [inline ai-chatbot-ampere/lab4_deploy_application/lab4_deploy_application.md:1]: alt text needed on images

File: ai-chatbot-ampere/lab5_interact_with_chatbot/lab5_interact_with_chatbot.md
Comment (hope-fisher) [inline ai-chatbot-ampere/lab5_interact_with_chatbot/lab5_interact_with_chatbot.md:1]: alt text needed on images

**Review by hope-fisher — COMMENTED** [review]
alt text needed on images

**Comment by animeshsahay** [comment]
Discussed internally about the alt texts

**Review by hope-fisher — APPROVED** [review]

**Review by hope-fisher — APPROVED** [review]

**Pull request merged**